scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -4,6 +4,8 @@ import "GvsWarpTasks.wdl" as Tasks
 import "GvsUtils.wdl" as Utils
 import "../../vcf_site_level_filtering_wdl/JointVcfFiltering.wdl" as VQSRLite
 
+# This is a reminder to set the priors back to non-zero in GvsWarpTasks before you merge this PR!!  TODO
+
```

Comment:
Can this be done with a passed argument?

Reply:
Yeah, that'd be a good idea. I'm still working with Sam to make sure I run classic in the way he wants in order to remove YNG issues. So if what I've done here isn't the way (which I think is what's happening) I may remove this all - but I will try to parameterize it for future work.